### ISSUE TITLE
Getting a 404- Not Found page.

### ISSUE THREAD

**Issue by grshankersai** [omoponfhir/omoponfhir-main-r4-sql#7]
Hello, I followed all the instructions to set up the omoponfhir project. But when I run it using docker, I am getting the 404-Not Found error page. Tried to resolve it in many ways, but not able to figure out where I am going wrong  Can someone kindly help me resolve it?

**Comment by myungchoi** [comment]
Please post the url that you used to browse the page.

**Comment by grshankersai** [comment]
Hi @myungchoi ,
Thank you so much for your response, I was able to set the project up, Able to see the home page.
The problem I am facing now is, I have the Json FHIR data for a patient and also the DB set up, I am unable to figure out the which api call to make so that the JSON data is converted and stored in the DB.
Could you please help me in that?
Currently, I am trying with the /smart end point which shows the Home page. unable to figure out what to do next.

Thank you!

**Comment by myungchoi** [comment]
Test Page : http(s)://<your omoponfhir server>/FHIR API endpoint: http(s)://<your omoponfhir server>/fhirSMART on FHIR app registration page: http(s)://<your omoponfhir server>/smartPlease try these On Nov 17, 2022, at 1:45 AM, grshankersai ***@***.***> wrote:﻿
Hi @myungchoi ,
Thank you so much for your response, I was able to set the project up, Able to see the home page.
The problem I am facing now is, I have the Json FHIR data for a patient and also the DB set up, I am unable to figure out the which api call to make so that the JSON data is converted and stored in the DB.
Could you please help me in that?
Currently, I am trying with the /smart end point which shows the Home page. unable to figure out what to do next.
Thank you!

**Comment by grshankersai** [comment]
Thank you @myungchoi , Able to make figure out things now! Thank you so much for the help, One last question, I was able to work with the patient resource type. but was not able to find the Explanation of benefit resource type in the project.
Is Explanation of benefit resource type under development?

**Comment by myungchoi** [comment]
No... sorry. That is not on our roadmap yet. Do you have mapping info for
that?

**Comment by grshankersai** [comment]
Yes @myungchoi , Currently I am looking to map the Explanation of Benefit resource type,
Mapping info for that :
https://hl7.org/fhir/explanationofbenefit.html - Docs
